src/components/Heading/Heading.tsx

```diff
@@ -24,24 +31,33 @@ export interface HeadingProps {
   urlClass?: string;
 }
 
-export default function Heading(props: React.PropsWithChildren<HeadingProps>) {
+// Used to map between ButtonTypes enum values and Chakra variant options.
+const variantMap = {};
+for (const type in HeadingDisplaySizes) {
+  variantMap[HeadingDisplaySizes[type]] = HeadingDisplaySizes[type];
+}
+
+/**
+ * Map the HeadingDisplaySizes to the Heading Chakra theme variant object. If a wrong
+ * value is passed (typically in non-Typescript scenarios), then the default
+ * is the "primary" variant.
```

Comment:
Technically, the default is `null` in this case.

Reply:
Fixed.